README.md

```diff
@@ -13,7 +13,7 @@ First of all, you can install this pack as below
 $ st2 pack install zabbix
 ```
```

Comment:
Should update the screenshots in here with the new parameters for the dispatch script.

Reply:
I updated screenshot and descriptions for the `st2_dispatch.py` by bf6777e and 809cb09.